src/utils.ts

```diff
@@ -160,15 +160,30 @@ export const removeSelection = () => {
 
 export const distance = (x: number, y: number) => Math.abs(x - y);
 
-export const resetCursor = () => {
-  document.documentElement.style.cursor = "";
+export const resetCursor = (canvas: HTMLCanvasElement | null) => {
+  if (!canvas) {
+    return;
+  }
+  canvas.style.cursor = "";
+};
+
+export const setCursor = (canvas: HTMLCanvasElement | null, cursor: string) => {
+  if (canvas) {
+    canvas.style.cursor = cursor;
+  }
```

Comment:
```suggestion
  canvas?.style.cursor = cursor;
```

Reply:
sadly, that's illegal in JS: it'd try to assign `cursor` to `undefined` in case `canvas` was null